platform/android/export/export.cpp

```diff
@@ -845,12 +841,17 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		Vector<String> perms;
 		_get_permissions(p_preset, p_give_internet, perms);
 		for (int i = 0; i < perms.size(); i++) {
-			manifest_text += vformat("    <uses-permission android:name=\"%s\" />\n", perms.get(i));
+			String permission = perms.get(i);
+			if (permission == "android.permission.WRITE_EXTERNAL_STORAGE" || permission == "android.permission.READ_EXTERNAL_STORAGE") {
+				manifest_text += vformat("    <uses-permission android:name=\"%s\" android:maxSdkVersion=\"28\" />\n", permission);
```

Comment:
IIUC, SDK 28 is the last one which allowed using this, and SDK 29 allowed it if `requestLegacyExternalStorage` is defined?

Reply:
That's correct. I'll also leave the `requestLegacyExternalStorage` flag in for now as well.